src/components/AssetSelector.tsx

```diff
@@ -32,32 +25,26 @@ export function AssetSelectorItem({
   onClick
 }: ItemProps) {
   return (
-    <SContainer
+    <Box
+      display="flex"
+      alignItems="center"
+      flexDirection="row"
+      padding="11px 10px"
       {...(onClick ? { onPointerDown: onClick } : null)}
       data-testid={`asset-selector-option-${ticker}`}
     >
       {showAssetIcon && <AssetIcon uuid={uuid} size={'1.5rem'} />}
       <Typography
-        bold={true}
```

Comment:
Why the bold change?

Reply:
I assume we have it bold to match the design of other Selectors. Fx. see Send Assets, where account names are bold

Reply:
It seems that the bold is an implementation error so I corrected it.
If we look at the design specs the account names should also be 'normal'
https://www.figma.com/file/vsmfxvOp3JFKTEXdsSIFWI/MyCrypto-Web-App-Working-File?node-id=1257%3A89144